pkg/traceql/ast.go

```diff
@@ -1073,6 +1081,11 @@ func (a *MetricsAggregate) init(q *tempopb.QueryRangeRequest, mode AggregateMode
 		exemplarFn = func(s Span) (float64, uint64) {
 			return math.NaN(), a.spanStartTimeMs(s)
 		}
+	case metricsAggregateMinOverTime:
+		innerAgg = func() VectorAggregator { return NewMinOverTimeAggregator(a.attr) }
+		exemplarFn = func(s Span) (float64, uint64) {
```

Comment:
I am not quite sure if the exemplar function is correct in this case

Reply:
It should use the real attribute value from the floatize function, so that the exemplar is plotted in the final graph like expected.  math.NaN is being used for rate and count_over_time because they don't come from an attribute, it's a placeholder and the value gets assigned later.

Reply:
Yes, it is. But the attr.Float() may return a NaN so we need to prevent the error